frontend/src/assets/scss/bt/catalog/_modal.scss

```diff
@@ -28,7 +28,7 @@
     .modal-content {
       border: 0px;
       border-radius: 0px;
-      overflow: scroll;
```

Comment:
I don't think we want to disable scroll here since this controls the modal on mobile views for the Catalog page.